packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -84,6 +84,55 @@ const ShowNotifications = ({ is_notifications_visible, notifications_count, togg
     );
 };
 
+const DefaultMobileLinks = ({
+    is_dark_mode,
+    toggleIsTourOpen,
+    is_mf,
+    is_eu,
+    is_eu_country,
+    setIsOnboardingVisited,
+    is_notifications_visible,
+    notifications_count,
+    toggleNotifications,
+    handleClickCashier,
```

Comment:
@shaheer-deriv I think you can get most of them directly from the store instead of prop drilling 🤔

Reply:
on it.. 👍🏼